build.gradle

```diff
@@ -38,7 +35,7 @@ dependencies {
     compile "redis.clients:jedis:2.8.0"
     testCompile "org.easymock:easymock:3.4"
     testCompile 'junit:junit:[4.10,)'
-    testRuntime "org.slf4j:slf4j-simple:1.7.7"
+    testRuntime "ch.qos.logback:logback-classic:1.1.3"
```

Comment:
Using logback for tests allows for more helpful log messages that include source code  line number.